# ucr:
# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Wheel & Steer
steer_drive_controller:
  type                          : "steer_drive_controller/SteerDriveController"
  rear_wheel                    : 'rear_drive_joint'
  front_steer                   : 'front_steer_joint'
  publish_rate                  : 30

  pose_covariance_diagonal      : [0.00001, 0.00001, 1000000000000.0, 1000000000000.0, 1000000000000.0, 0.001]
  twist_covariance_diagonal     : [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  wheel_radius_multiplier       : 0.1047 # ratio of rad/s v.s. rmp
  steer_pos_multiplier          : 318.31 # ratio of rad v.s. qc(2000qc/round for current encoder)
  base_frame_id                 : base_footprint

  enable_odom_tf: true
  wheel_separation_h            : 0.368
  wheel_radius                  : 0.0765
